Remove useless check at build time

At build time, Qwt looks whether the Qwt libraries are already
installed in the destination directory, to avoid confusion between
libraries installed system-wide and locally-built shared
libraries. However, this behaviour conflicts with how Buildroot builds
and installs libraries, so get rid of it.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: qwt-6.0.0/qwtbuild.pri
===================================================================
--- qwt-6.0.0.orig/qwtbuild.pri	2011-09-12 19:37:35.663275952 +0200
+++ qwt-6.0.0/qwtbuild.pri	2011-09-12 19:37:41.913275949 +0200
@@ -53,17 +53,3 @@
 !debug_and_release {
     OBJECTS_DIR       = obj
 }
-
-unix {
-
-    exists( $${QMAKE_LIBDIR_QT}/libqwt.* ) {
-
-        # On some Linux distributions the Qwt libraries are installed 
-        # in the same directory as the Qt libraries. Unfortunately
-        # qmake always adds QMAKE_LIBDIR_QT at the beginning of the 
-        # linker path, so that the installed libraries will be
-        # used instead of the local ones.
-
-        error( "local build will conflict with $${QMAKE_LIBDIR_QT}/libqwt.*" )
-    }
-}
